docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -122,3 +122,22 @@ SQL support
 The connector provides :ref:`globally available <sql-globally-available>` and
 :ref:`read operation <sql-read-operations>` statements to access data and
 metadata in Pinot.
+
+.. _pinot-pushdown:
+
+Pushdown
+--------
+
```

Comment:
I am not sure if limit, join or topn pushdown are supported.. I can add if you can confirm @elonazoulay

Reply:
Limit pushdown seems implemented. Join and topn are not.

Reply:
Updated